llvm/include/llvm/ADT/PackedVector.h

```diff
@@ -141,6 +141,8 @@ class PackedVector : public PackedVectorBase<T, BitNum, BitVectorTy,
     Bits |= RHS.Bits;
     return *this;
   }
+
+  operator BitVectorTy() const { return Bits; }
```

Comment:
Isn't this copying the underlying vector each time you want access?

How about providing reference access to it instead, maybe something like
```
const BitVectorTy &raw_bits() const { return Bits; }
```
?

Reply:
That's even easier to use. Done.